UPGRADE-2.0.md

```diff
@@ -8,3 +8,7 @@ This is the list of actions that you need to take when upgrading this bundle fro
 composer remove --dev nelmio/alice
 composer require --dev liip/functional-test-bundle "~2.0"
 ```
+
+ * It had been changed the interface of `LoadFixtures` to allow append fixtures. The main difference is it had been added
```

Comment:
just a small wording tweak

`The interface of `LoadFixtures` had to be changed to allow append fixtures.`

Reply:
@lsmith77 Thanks, it is sorted.